prow/plugins/verify-owners/verify-owners.go

```diff
@@ -129,7 +129,7 @@ func handle(ghc githubClient, gc *git.Client, roc repoownersClient, log *logrus.
 	// List modified OWNERS files.
 	var modifiedOwnersFiles []github.PullRequestChange
 	for _, change := range changes {
-		if filepath.Base(change.Filename) == ownersFileName {
+		if filepath.Base(change.Filename) == ownersFileName && change.Status != github.PullRequestFileRemoved {
```

Comment:
I guess it'll also be better if we added this to OWNERS_ALIASES file check below as well. Rare to have that file removed, but you never know. :)

Reply:
:+1:, will fix it there too!

Reply:
I have looked into this case, and the current implementation handles the `OWNERS_ALIASES` removal quite fine (I have added a testcase that executes it): it does not attempt to parse that file if it does not exist, and the flow treats it like "aliases changed and now we have an empty set of aliases".

This behavior is IMHO worth keeping: compared to removing `OWNERS` (which never results in invalid state), removing aliases may result in something being really invalid, like if you remove an alias but continue to use it in unchanged `OWNERS` file. Currently, the plugin does not check for this case at all and implementing that check is IMO beyond the scope of this PR.

Sounds good?

Reply:
Sgtm :+1: